python/lsst/sims/maf/slicers/baseSpatialSlicer.py

```diff
@@ -59,7 +60,10 @@ def __init__(self, verbose=True, lonCol='fieldRA', latCol='fieldDec',
         self.slicePoints['ra'] = None
         self.slicePoints['dec'] = None
         self.nslice = None
+        self.shape = None
         self.plotFuncs = [BaseHistogram, BaseSkyMap]
+        # Values in self.slicePoints to pass whole to the metrics
+        self.bulkSlicePointKeys = []
```

Comment:
Oh - I was just wondering something. 
The metrics that work with these 2d slicers are the 'vectorMetrics'. 
But otherwise, the slicers are just the same as the regular slicers of the similar name, right? (the slicer.shape changes and the binpoints have to be set, and the plotter changed, but otherwise the same). 
I wonder if there's a way to make this simpler, make these values (slicer.shape, binpoints, plotter values) be set or updated in a method on BaseSlicer, so that any slicer *could* be a vectorSlicer with these small modifications. Then, depending on what *metric* was associated with the slicer in the metricBundle, the slicer could become a vectorSlicer or could remain a regular slicer. 
??